srv/pillar/ceph/disk_led.sls

```diff
@@ -0,0 +1,15 @@
+# This is the default configuration for the storage enclosure LED blinking.
+# The placeholder {device_file} will be replaced with the device file of
+# the disk when the command is executed.
+#
+# Have a look into the README file to find out how to customize this
```

Comment:
Maybe say `/srv/pillar/ceph/README` rather than 'the README file'?

Reply:
Done